martin/src/styles/mod.rs

```diff
@@ -106,6 +106,7 @@ impl StyleSources {
             Ok(contained_paths) => {
                 for child_path in contained_paths {
                     let Some(name) = child_path.file_name() else {
+                        // FIXME: we should never panic in production code
```

Comment:
this assertion is here, as it logically cannot happen.

The base path is checked if it is a valid file name one level up (where the id is extracted).